test/test_utils.py

```diff
@@ -41,6 +43,27 @@ def test_remove_samples_with_label(df, sample_to_label, label_to_remove, correct
         assert test_column == correct_column
 
 
+class1_encoding = np.zeros((3, 3))
+class1_encoding[0, 0] = 1
+class2_encoding = np.zeros((3, 3))
+class2_encoding[1, 1] = 1
+class3_encoding = np.zeros((3, 3))
+class3_encoding[2, 2] = 1
+
+
+@pytest.mark.parametrize('classes,correct_encodings',
+                         [(['class1', 'class2', 'class3'],
+                           [class1_encoding, class2_encoding, class3_encoding]),
+                          (['class1', 'class2'], [0, 1]),
+                          ])
+def test_generate_encoding(classes, correct_encodings):
```

Comment:
Maybe add a comment for what you are testing here.

Reply:
Sounds good